### ISSUE TITLE
Cannot parse headers

### ISSUE THREAD

**Issue by pontaoski** [mini2Dx/gettext#10]
```
org.antlr.v4.runtime.misc.ParseCancellationException: Error parsing .po file. Line 5:1 - no viable alternative at input '#\n'
```

Example header:
```po
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
```

**Comment by tomcashman** [comment]
I'll have some time this evening to investigate and patch

**Comment by tomcashman** [comment]
Fixed in 1.11.0 with is released now.
